### PR TITLE
Revert "Revert "Add files via upload" (#2)"

### PR DESCRIPTION
**Pull request by ar-reddy**
This reverts commit a7de95d03f450e1f74e28d6d237170813184e971.

@JatVineet999  created the PR on behalf of you please check the comments below

### CODE REVIEW

File: home.html [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+    <link
+      href="https://fonts.googleapis.com/css2?family=Nunito&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500;600;700;800&display=swap"
+      rel="stylesheet"
+    />
```
Comment (ar-reddy) [inline home.html]: clean the URL import fonts inside css file

File: home.html [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+          />
+        </section>
+        <section class="nav-items">
+          <ul>
+            <li>Destinations</li>
+            <li>Hotels</li>
+            <li>Flights</li>
+            <li>Bookings</li>
+            <li>Login</li>
+            <li class="signup-btn">Sign up</li>
```
Comment (ar-reddy) [inline home.html]: i think signup is a button why u have used li

create common class for button u have to reuse

File: home.html [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+            <li class="language-dropdown" style="font-size: smaller">
```
Comment (ar-reddy) [inline home.html]: if u have class name why u have added the inline styles here?

File: home.html [hunk 4]
```diff
@@ -0,0 +1,327 @@
...
+          <h1>
+            Travel, enjoy <br />and live a new <br />
+            and full life
+          </h1>
+          <p>
+            Built Wicket Longer admire do barton vanity itself do in it. <br />
+            Preferred to sportsmen it engrossed listening. Park gate <br />
+            sell they west hard for the.
+          </p>
+          <button class="cta-button">Find out more</button>
```
Comment (ar-reddy) [inline home.html]: class name should be specific to that button, and u have to create the common class name for button and reuse that

File: home.html [hunk 5]
```diff
@@ -0,0 +1,327 @@
...
+        </span>
+        <span class="right-section">
+          <img src="elements\traveller.png" alt="" />
```
Comment (ar-reddy) [inline home.html]: if the alt is empty no need to specify, alt has some usage please check and update the code

File: home.html [hunk 6]
```diff
@@ -0,0 +1,327 @@
...
+    </header>
+    <section class="category">
+      <div>
+        <header>CATEGORY</header>
+        <section class="category-content">We Offer Best Services</section>
+      </div>
+      <img
+        src="elements\plus.png"
+        alt=""
+        style="position: absolute; transform: translate(645px, 20%)"
```
Comment (ar-reddy) [inline home.html]: inline styles should be removed

File: home.html [hunk 7]
```diff
@@ -0,0 +1,327 @@
...
+          <button class="cta-button">Find out more</button>
+        </span>
+        <span class="right-section">
+          <img src="elements\traveller.png" alt="" />
+        </span>
+      </main>
+    </header>
+    <section class="category">
+      <div>
+        <header>CATEGORY</header>
```
Comment (ar-reddy) [inline home.html]: header tag should not use in section header has different meaning like footer

File: home.html [hunk 8]
```diff
@@ -0,0 +1,327 @@
...
+        src="elements\plus.png"
+        alt=""
+        style="position: absolute; transform: translate(645px, 20%)"
+      />
+    </section>
+
+    <div class="services-list">
+      <div class="service-card">
+        <div class="card-img">
+          <img src="elements\weather.png" alt="Weather Image" />
```
Comment (ar-reddy) [inline home.html]:
```suggestion
          <img src="images\weather.png" alt="Weather Image" />

```

File: home.html [hunk 9]
```diff
@@ -0,0 +1,327 @@
...
+          <h2>Local Events</h2>
+          <p>
+            Barton vanity itself do in it. Preferred to men it engrossed
+            listening.
+          </p>
+        </div>
+      </div>
+      <div class="service-card">
+        <div class="card-img">
+          <img src="elements\customization.png" alt="Customization Image" />
```
Comment (ar-reddy) [inline home.html]:
```suggestion
          <img src="images\customization.png" alt="Customization Image" />

```

for all images the changes should be made

File: home.html [hunk 10]
```diff
@@ -0,0 +1,327 @@
...
+          <p>We deliver outsourced aviation services for military customers.</p>
+        </div>
+      </div>
+    </div>
+    <section class="top-selling">
+      <header>
+        <div><h4>Top selling</h4></div>
+        <div><h1>Top Destinations</h1></div>
+      </header>
+      <main class="top-selling-list">
```
Comment (ar-reddy) [inline home.html]: The ```<header>``` element has an identical meaning to the site-wide banner landmark role, unless nested within sectioning content. Then, the <header> element is not a landmark. The <header> element can define a global site header, described as a banner in the accessibility tree.

File: home.html [hunk 11]
```diff
@@ -0,0 +1,327 @@
...
+              <h4>Reach Airport on Selected Date</h4>
+              <p class="para">
+                Lorem ipsum dolor sit amet, consectetur adipisicing
+                elit.Urna,tortor tempus.
+              </p>
+            </div>
+          </section>
+        </section>
+      </section>
+      <section class="right">
```
Comment (ar-reddy) [inline home.html]: don't use right left for class name keep meaning full names

File: home.html [hunk 12]
```diff
@@ -0,0 +1,327 @@
...
+              <img src="elements/map.png" alt="" style="width: 25%" />
+              <img src="elements/send.png" alt="send-icon" />
+            </div>
+            <div class="trip-days">
+              <img src="elements\building.png" />
+              <p>24 people going</p>
+              <img
+                src="elements/heart (6) 1.png"
+                alt="like"
+                style="margin-left: 34%; width: 5%"
```
Comment (ar-reddy) [inline home.html]: in this you have used lot of inline styles please decrease as many as

File: home.html [hunk 13]
```diff
@@ -0,0 +1,327 @@
...
+        </div>
+        <section class="overhead-banner">
+          <section class="card">
+            <section class="aside-image">
+              <img src="elements/rome-sm.png" alt="rome" />
+            </section>
+            <section class="aside-text">
+              <h6 style="margin-top: 20px">Ongoing</h6>
+              <h4 style="margin: 0%">Trip to rome</h4>
+              <h6 style="margin-top: 8px; display: inline; color: #b4a7f8">
```
Comment (ar-reddy) [inline home.html]: same here

File: home.html [hunk 14]
```diff
@@ -0,0 +1,327 @@
...
+              <div class="progress-line">
+                <div class="completed-bar"></div>
+              </div>
+            </section>
+          </section>
+        </section>
+      </section>
+    </section>
+    <div class="sub-container">
+      <div class="telegram"><img src="elements/Group 77.png" alt="tele" /></div>
```
Comment (ar-reddy) [inline home.html]: image names should be meaning full

File: home.html [hunk 15]
```diff
@@ -0,0 +1,327 @@
...
+        <div class="rings"><img src="elements/Group 42.png" alt="rings" /></div>
+        <div class="right-r">
+          <img src="elements/Group 42.png" alt="rings" />
+        </div>
+        <section class="heading">
+          Subscribe to get information, latest news and other <br />
+          interesting offers about Cobham
+        </section>
+        <section class="mailer">
+          <div class="email-typer">
```
Comment (ar-reddy) [inline home.html]:
```suggestion
          <div class="email-type">

```

File: home.html [hunk 16]
```diff
@@ -0,0 +1,327 @@
...
+      <section class="subscribe">
+        <div class="rings"><img src="elements/Group 42.png" alt="rings" /></div>
+        <div class="right-r">
+          <img src="elements/Group 42.png" alt="rings" />
+        </div>
+        <section class="heading">
+          Subscribe to get information, latest news and other <br />
+          interesting offers about Cobham
+        </section>
+        <section class="mailer">
```
Comment (ar-reddy) [inline home.html]:
```suggestion
        <section class="email">

```

File: home.html [hunk 17]
```diff
@@ -0,0 +1,327 @@
...
+              type="email"
+              placeholder="Your email"
+            />
+          </div>
+          <div class="Subscribe-btn">Subscribe</div>
+        </section>
+      </section>
+    </div>
+    <footer class="footer">
+      <div class="f1">
```
Comment (ar-reddy) [inline home.html]: don't use f1 as a class name

File: home.html [hunk 18]
```diff
@@ -0,0 +1,327 @@
...
+        <h1>TravelPro</h1>
+        <p>
+          Book your trip in a minute,get full <br />
+          control for much longer.
+        </p>
+      </div>
+      <div class="about">
+        <h3>Company</h3>
+        <h5><a href="www.google.com">About</a></h5>
```
Comment (ar-reddy) [inline home.html]:
```suggestion
        <h5><a href="#">About</a></h5>

```

Check why we need to keep # if we don't know the route and make these changes for below as well

File: style.css [hunk 19]
```diff
@@ -0,0 +1,657 @@
...
+}
+
+nav ul {
+  list-style-type: none;
+  display: flex;
+  margin-right: 50px;
+  padding-right: 80px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```
Comment (ar-reddy) [inline style.css]: remove this and check once, any way parent has the same css properties

File: style.css [hunk 20]
```diff
@@ -0,0 +1,657 @@
...
+
+nav ul .language-dropdown .dropdown-content li:last-child {
+  border-bottom: none;
+}
+main {
+  display: flex;
+  margin-bottom: 200px;
+}
+
+.left-section {
```
Comment (ar-reddy) [inline style.css]: name should be meaningful related to that section

File: style.css [hunk 21]
```diff
@@ -0,0 +1,657 @@
...
+  padding: 0 0 0 8%;
+  margin: 0% 0% 0% 7%;
+  position: relative;
+  max-width: 35%;
+}
+
+.right-section img {
```
Comment (ar-reddy) [inline style.css]: same here

File: style.css [hunk 22]
```diff
@@ -0,0 +1,657 @@
...
+  left: 39em;
+  z-index: 1;
+  width: 40%;
+  margin-right: 50%;
+}
+main span h5 {
+  margin-top: 17%;
+  margin-bottom: 0%;
+  color: #e27660;
+  font-family: "Poppins", sans-serif;
```
Comment (ar-reddy) [inline style.css]: for complete website we are using same font family don't repeat, you can apply the font for body or u can take container class and specify there

File: style.css [hunk 23]
```diff
@@ -0,0 +1,657 @@
...
+  font-weight: 600;
+}
+main span h1 {
+  font-size: 400%;
+  line-height: 1;
+  font-weight: 700;
+  font-family: "Poppins", sans-serif;
```
Comment (ar-reddy) [inline style.css]: same here

File: style.css [hunk 24]
```diff
@@ -0,0 +1,657 @@
...
+  padding: 15px 25px;
+  font-size: medium;
+  background-color: #f1a501;
+  color: #fff;
+  border: none;
+  border-radius: 10px;
+  cursor: pointer;
+  transition: background-color 0.3s ease;
+}
```
Comment (ar-reddy) [inline style.css]: button should have separate css and reuse that

File: home.html (hunk 1 above)
Comment (ar-reddy) [reply home.html]: > clean the URL import fonts inside css file

@JatVineet999 Please check the reviews

**Comment by vijith07** [comment]
Explore what HTML semantics are. Understand them and use them in your code. Decrease the usage of div everywhere. Use appropriate tags.
https://web.dev/learn/html/semantic-html

File: home.html (hunk 1 above)
Comment (JatVineet999) [reply home.html]: Done

File: home.html (hunk 2 above)
Comment (JatVineet999) [reply home.html]: Resolved, My bad

File: home.html (hunk 3 above)
Comment (JatVineet999) [reply home.html]: resolved

File: style.css (hunk 20 above)
Comment (JatVineet999) [reply style.css]: resloved

File: style.css (hunk 21 above)
Comment (JatVineet999) [reply style.css]: resolved

File: style.css (hunk 22 above)
Comment (JatVineet999) [reply style.css]: resolved

File: style.css (hunk 23 above)
Comment (JatVineet999) [reply style.css]: resolved

File: home.html (hunk 7 above)
Comment (JatVineet999) [reply home.html]: resolved

File: home.html (hunk 8 above)
Comment (JatVineet999) [reply home.html]: resolved

File: home.html (hunk 9 above)
Comment (JatVineet999) [reply home.html]: done

File: home.html (hunk 10 above)
Comment (JatVineet999) [reply home.html]: resolved

File: home.html (hunk 12 above)
Comment (JatVineet999) [reply home.html]: all resolved only 2 left.

File: home.html (hunk 13 above)
Comment (JatVineet999) [reply home.html]: resolved

File: home.html (hunk 17 above)
Comment (JatVineet999) [reply home.html]: resolved

File: home.html (hunk 18 above)
Comment (JatVineet999) [reply home.html]: resolved

File: style.css (hunk 19 above)
Comment (JatVineet999) [reply style.css]: resolved

File: home.html (hunk 5 above)
Comment (JatVineet999) [reply home.html]: resolved

File: home.html (hunk 14 above)
Comment (JatVineet999) [reply home.html]: resolved

File: home.html (hunk 15 above)
Comment (JatVineet999) [reply home.html]: resolved

File: home.html (hunk 16 above)
Comment (JatVineet999) [reply home.html]: resolved

File: style.css (hunk 24 above)
Comment (JatVineet999) [reply style.css]: resolved

File: home.html (hunk 12 above)
Comment (JatVineet999) [reply home.html]: all resolved

File: Index.html [hunk 25]
```diff
@@ -0,0 +1,303 @@
...
+              <span class="dropdown-trigger">EN&#9660;</span>
+              <ul class="dropdown-content">
+                <li>Hindi</li>
+                <li>German</li>
+              </ul>
+            </div>
+          </ul>
+        </section>
+      </nav>
+      <main>
```
Comment (tarunkumarpk) [inline Index.html]: Main content shouldn't be placed inside `header` tag.
move `main` tag to outside. Place it after `header` tag

File: Index.html [hunk 26]
```diff
@@ -0,0 +1,303 @@
...
+          <ul>
+            <li>Destinations</li>
+            <li>Hotels</li>
+            <li>Flights</li>
+            <li>Bookings</li>
+            <li>Login</li>
+            <div class="signup-btn">Sign up</div>
+            <div class="language-dropdown">
+              <span class="dropdown-trigger">EN&#9660;</span>
+              <ul class="dropdown-content">
```
Comment (tarunkumarpk) [inline Index.html]: use `select` tag for dropdown

File: Index.html [hunk 27]
```diff
@@ -0,0 +1,303 @@
...
+    </header>
+    <section class="category">
+      <div>
+        <div class="category">CATEGORY</div>
+        <section class="category-content">We Offer Best Services</section>
+      </div>
+      <img src="images\plus.png" alt="plus" />
+    </section>
+
+    <div class="services-list">
```
Comment (tarunkumarpk) [inline Index.html]: `<div class="services-list">`
use `section` tag

File: Index.html [hunk 28]
```diff
@@ -0,0 +1,303 @@
...
+          <img src="images\customization.png" alt="Customization Image" />
+        </div>
+        <div class="card-content">
+          <h2>Customization</h2>
+          <p>We deliver outsourced aviation services for military customers.</p>
+        </div>
+      </div>
+    </div>
+    <section class="top-selling">
+      <div class="selling-header">
```
Comment (tarunkumarpk) [inline Index.html]: use `header` tag for defining header in section

File: Index.html [hunk 29]
```diff
@@ -0,0 +1,303 @@
...
+        <div><h4>Top selling</h4></div>
+        <div><h1>Top Destinations</h1></div>
+      </div>
+      <main class="top-selling-list">
```
Comment (tarunkumarpk) [inline Index.html]:
``` suggestion
    <div class="top-selling-list">
```

File: Index.html [hunk 30]
```diff
@@ -0,0 +1,303 @@
...
+        <div class="top-selling-card">
```
Comment (tarunkumarpk) [inline Index.html]:
```suggestion
         <article class="top-selling-card">
```

File: Index.html [hunk 31]
```diff
@@ -0,0 +1,303 @@
...
+          <div class="destination-details">
+            <p>Rome, Italy</p>
+            <p>$5,42k</p>
+          </div>
+          <div class="trip-days">
+            <img src="images\send-icon.png" />
+            <p>10 Days Trip</p>
+          </div>
+        </div>
+        <div class="top-selling-card">
```
Comment (tarunkumarpk) [inline Index.html]: change here also to article

File: Index.html [hunk 32]
```diff
@@ -0,0 +1,303 @@
...
+          <div class="destination-details">
+            <p>London, UK</p>
+            <p>$4.2k</p>
+          </div>
+          <div class="trip-days">
+            <img src="images\send-icon.png" />
+            <p>12 Days Trip</p>
+          </div>
+        </div>
+        <div class="top-selling-card">
```
Comment (tarunkumarpk) [inline Index.html]: change here also to article tag

File: Index.html (hunk 25 above)
Comment (JatVineet999) [reply Index.html]: resolved, thanks for the input.

File: Index.html (hunk 26 above)
Comment (JatVineet999) [reply Index.html]: resolved

File: Index.html (hunk 28 above)
Comment (JatVineet999) [reply Index.html]: resolved

File: Index.html (hunk 30 above)
Comment (JatVineet999) [reply Index.html]: resolved

File: Index.html [hunk 33]
```diff
@@ -0,0 +1,303 @@
...
+              type="email"
+              placeholder="Your email"
+            />
+          </div>
+          <div class="Subscribe-btn">Subscribe</div>
+        </section>
+      </section>
+    </div>
+    <footer class="footer">
+      <div class="footer-part1">
```
Comment (tarunkumarpk) [inline Index.html]:
```suggestion
      <section class="logo-and-slogan">
```

File: Index.html [hunk 34]
```diff
@@ -0,0 +1,303 @@
...
+          </div>
+        </div>
+        <div class="discover"><h3>Discover our app</h3></div>
+        <div class="download">
+          <div><img src="images/gpay-icon.png" alt="gpay" /></div>
+          <div><img src="images/apple-store.png" alt="apple-store" /></div>
+        </div>
+      </div>
+    </footer>
+    <div class="rights">All rights reserved@travelpro.com</div>
```
Comment (tarunkumarpk) [inline Index.html]:
```suggestion
 <div class="rights">All rights reserved@travelpro.com</div>
    </footer>
```

File: Index.html [hunk 35]
```diff
@@ -0,0 +1,303 @@
...
+    </div>
+    <footer class="footer">
+      <div class="footer-part1">
+        <h1>TravelPro</h1>
+        <p>
+          Book your trip in a minute,get full <br />
+          control for much longer.
+        </p>
+      </div>
+      <div class="about">
```
Comment (tarunkumarpk) [inline Index.html]:
```suggestion
      <nav class="footer-nav">

```

File: Index.html [hunk 36]
```diff
@@ -0,0 +1,303 @@
...
+        <h3>Company</h3>
+        <h5><a href="#">About</a></h5>
+        <h5><a href="#">Careers</a></h5>
+        <h5><a href="#">Mobile</a></h5>
+      </div>
+      <div class="about">
```
Comment (tarunkumarpk) [inline Index.html]:
```suggestion
      <nav class="footer-nav">

```

File: Index.html [hunk 37]
```diff
@@ -0,0 +1,303 @@
...
+        <h3>Contact</h3>
+        <h5><a href="#">Help/FAQ</a></h5>
+        <h5><a href="#">Press</a></h5>
+        <h5><a href="#">Affilates</a></h5>
+      </div>
+      <div class="about">
```
Comment (tarunkumarpk) [inline Index.html]:
```suggestion
      <nav class="footer-nav">

```

File: Index.html [hunk 38]
```diff
@@ -0,0 +1,303 @@
...
+        <h3>More</h3>
+        <h5><a href="#">Airlinefees</a></h5>
+        <h5><a href="#">Airline</a></h5>
+        <h5><a href="#">Low Fair Tips</a></h5>
+      </div>
+      <div class="social-links">
```
Comment (tarunkumarpk) [inline Index.html]:
```suggestion
      <section class="social-links">

```

File: Index.html [hunk 39]
```diff
@@ -0,0 +1,303 @@
...
+              <h6 class="completion">completed</h6>
+              <div class="progress-line">
+                <div class="completed-bar"></div>
+              </div>
+            </section>
+          </section>
+        </section>
+      </section>
+    </section>
+    <div class="sub-container">
```
Comment (tarunkumarpk) [inline Index.html]:
```suggestion
    <section class="sub-container">

```

**Comment by tarunkumarpk** [comment]
Refactor the HTML and CSS code by utilizing appropriate tags and class names.

File: Index.html (hunk 33 above)
Comment (JatVineet999) [reply Index.html]: done, thanks for the suggestion

File: Index.html (hunk 34 above)
Comment (JatVineet999) [reply Index.html]: done

File: Index.html (hunk 35 above)
Comment (JatVineet999) [reply Index.html]: done

File: Index.html [hunk 40]
```diff
@@ -0,0 +1,309 @@
...
+    <section class="main">
+      <div class="landing-left-section">
+        <h4>BEST DESTINATIONS AROUND THE WORLD</h4>
+        <h1>Travel, enjoy and live a new and full life</h1>
+        <p>
+          Built Wicket Longer admire do barton vanity itself do in it. <br />
+          Preferred to sportsmen it engrossed listening. Park gate <br />
+          sell they west hard for the.
+        </p>
+        <button class="btn cta-btn">Find out more</button>
```
Comment (ar-reddy) [inline Index.html]: the class name should starts with btn-

File: Index.html [hunk 41]
```diff
@@ -0,0 +1,309 @@
...
+        <img src="images/telegram.png" alt="telegram-icon" />
+      </div>
+      <div class="subscribe">
+        <div class="rings"><img src="images/rings.png" alt="rings" /></div>
+        <div class="right-r">
+          <img src="images/rings.png" alt="rings" />
+        </div>
+        <div class="heading">
+          Subscribe to get information, latest news and other <br />
+          interesting offers about Cobham
```
Comment (ar-reddy) [inline Index.html:250]: why u haven't used any html tags

File: Index.html [hunk 42]
```diff
@@ -0,0 +1,309 @@
...
+        </div>
+        <div class="email">
+          <div class="email-type">
+            <img src="images/email.png" alt="email" />
+            <input type="email" placeholder="Your email" />
+          </div>
+          <div class="btn Subscribe-btn">Subscribe</div>
```
Comment (ar-reddy) [inline Index.html]: for button u should not used div

File: Index.html [hunk 43]
```diff
@@ -0,0 +1,309 @@
...
+      </div>
+      <div class="landing-right-section">
+        <img src="images\traveller.png" alt="traveller" />
+      </div>
+    </section>
+
+    <section class="category">
+      <div class="category-container">
+        <div class="category-header">CATEGORY</div>
+        <section class="category-content">We Offer Best Services</section>
```
Comment (tarunkumarpk) [inline Index.html]: use `h` tags for headings, don't use `section` tag

File: Index.html [hunk 44]
```diff
@@ -0,0 +1,309 @@
...
+      </div>
+      <div>
+        <img src="images\plus.png" alt="plus" />
+      </div>
+    </section>
+
+    <section class="services-list">
```
Comment (tarunkumarpk) [inline Index.html:60]: move this section to `<section class="category">`
and update the class names also

File: Index.html [hunk 45]
```diff
@@ -0,0 +1,309 @@
...
+          <img src="images\customization.png" alt="Customization Image" />
+        </div>
+        <div class="card-content">
+          <h2>Customization</h2>
+          <p>We deliver outsourced aviation services for military customers.</p>
+        </div>
+      </div>
+    </section>
+    <section class="top-selling">
+      <div class="selling">
```
Comment (tarunkumarpk) [inline Index.html]:
```suggestion
      <header class="selling">

```

File: Index.html [hunk 46]
```diff
@@ -0,0 +1,309 @@
...
+        </article>
+      </div>
+    </section>
+    <section class="book-trip">
+      <div class="left">
+        <div class="book-trip-title"><p>Easy and Fast</p></div>
+        <div class="slogan">
+          <p>Book Your Next Trip <br />In 3 Easy Steps</p>
+        </div>
+        <div class="options">
```
Comment (tarunkumarpk) [inline Index.html]: use appropriate class names ex: `book-trip-steps` instead of `options`

File: Index.html [hunk 47]
```diff
@@ -0,0 +1,309 @@
...
+        </div>
+      </div>
+      <div class="right">
+        <div class="top-selling-card">
+          <div class="card-img">
+            <img src="images/girl.jpg" alt="Rome,Italy Image" />
+          </div>
+          <div class="card-texts">
+            <h3>Trip To Greece</h3>
+            <div class="music-player">
```
Comment (tarunkumarpk) [inline Index.html:203]: use appropriate class names ex: `trip-dates`

File: Index.html (hunk 40 above)
Comment (tarunkumarpk) [inline Index.html]: classname should be like `btn-cta`

File: Index.html [hunk 48]
```diff
@@ -0,0 +1,309 @@
...
+            <div class="trip-days">
+              <img src="images\send-icon.png" />
+              <p>28 Days Trip</p>
+            </div>
+          </div>
+        </article>
+      </div>
+    </section>
+    <section class="book-trip">
+      <div class="left">
```
Comment (tarunkumarpk) [inline Index.html]: use meaningful names for class

File: Index.html [hunk 49]
```diff
@@ -0,0 +1,309 @@
...
+              <img src="images/map.png" alt="map-icon" / class="map">
+              <img src="images/send.png" alt="send-icon" />
+            </div>
+            <div class="trip-days">
+              <img src="images\building.png" />
+              <p>24 people going</p>
+              <img src="images/heart (6) 1.png" alt="like" class="like" />
+            </div>
+          </div>
+          <div class="overhead-banner">
```
Comment (tarunkumarpk) [inline Index.html]: same here, use meaning full class names

**Comment by tarunkumarpk** [comment]
use meaning full class names and update in html and css.

File: Index.html [hunk 50]
```diff
@@ -0,0 +1,319 @@
...
+    <section class="category">
+      <div class="category-container">
+        <header class="category-header">CATEGORY</header>
+        <p class="category-content">We Offer Best Services</p>
+      </div>
+      <figure>
+        <img src="images\plus.png" alt="plus" />
+      </figure>
+    </section>
+    <section class="services-list">
```
Comment (tarunkumarpk) [inline Index.html:60]: services-list and category should be in one section

File: Index.html [hunk 51]
```diff
@@ -0,0 +1,319 @@
...
+          sell they west hard for the.
+        </p>
+        <button class="btn btn-cta">Find out more</button>
+      </div>
+      <div class="banner-right-section">
+        <img src="images\traveller.png" alt="traveller" />
+      </div>
+    </main>
+    <section class="category">
+      <div class="category-container">
```
Comment (tarunkumarpk) [inline Index.html]:
```suggestion
      <header class="category-container">

```

File: Index.html [hunk 52]
```diff
@@ -0,0 +1,319 @@
...
+        <header class="category-header">CATEGORY</header>
```
Comment (tarunkumarpk) [inline Index.html]: use h tags for headings

File: Index.html [hunk 53]
```diff
@@ -0,0 +1,319 @@
...
+              <img src="images\send-icon.png" />
+              <p>28 Days Trip</p>
+            </div>
+          </div>
+        </article>
+      </div>
+    </section>
+    <section class="book-trip">
+      <div class="book-trip-left">
+        <article class="book-trip-title">
```
Comment (tarunkumarpk) [inline Index.html]: use div tag

File: Index.html [hunk 54]
```diff
@@ -0,0 +1,319 @@
...
+          <p>Easy and Fast</p>
+        </article>
+        <article class="slogan">
```
Comment (tarunkumarpk) [inline Index.html]: same here

File: Index.html [hunk 55]
```diff
@@ -0,0 +1,319 @@
...
+        </div>
+      </div>
+      <section class="book-trip-right-section">
+        <article class="top-selling-card">
+          <figure class="card-img">
+            <img src="images/girl.jpg" alt="Rome, Italy Image" />
+          </figure>
+          <div class="trip-dates">
+            <h3>Trip To Greece</h3>
+            <article class="music-player">
```
Comment (tarunkumarpk) [inline Index.html]: same here

File: Index.html [hunk 56]
```diff
@@ -0,0 +1,319 @@
...
+              <p>24 people going</p>
+              <img src="images/heart.png" alt="like" class="like" />
+            </article>
+          </div>
+          <div class="aside-overhead-banner">
+            <div class="card">
+              <div class="aside-image">
+                <img src="images/rome-sm.png" alt="Rome" />
+              </div>
+              <article class="aside-text">
```
Comment (tarunkumarpk) [inline Index.html:222]: same here

File: Index.html [hunk 57]
```diff
@@ -0,0 +1,319 @@
...
+        </div>
+      </div>
+      <figure class="plus-r">
+        <img src="images\plus.png" alt="plus" />
+      </figure>
+    </section>
+    <footer class="footer">
+      <div class="footer-info">
+        <section class="logo-and-slogan">
+          <header>TravelPro</header>
```
Comment (tarunkumarpk) [inline Index.html:267]: use h tag

**Review by tarunkumarpk — APPROVED** [review]

**Pull request merged**